api/camunda/camunda-openapi.yaml

```diff
@@ -1935,7 +1935,7 @@ components:
         processDefinitionVersion:
           type: integer
           format: int32
-        bpmnProcessId:
+        processDefinitionId:
```

Comment:
:x: This is another actual field name change. I don't think this should be here, unless we're certain it's in the actual 8.6 api.